clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/data/task/jedis/JedisTaskRepository.groovy

```diff
@@ -124,6 +126,8 @@ class JedisTaskRepository implements TaskRepository {
       pipe.expire(resultId, TASK_TTL)
       pipe.sync()
     }
+
+    log.debug("addResultObjects (resultId: ${resultId}, values: ${values})")
```

Comment:
Unsure ... Chris was looking initially but from what I saw we add the result objects prior to flipping the task to complete.